sdk/python/kubeflow/storage_init_container/hugging_face.py

```diff
@@ -57,3 +59,37 @@ def download_model_and_tokenizer(self):
         transformers.AutoTokenizer.from_pretrained(
             self.model, cache_dir=self.config.download_dir
         )
+
+
+@dataclass
+class HfDatasetParams:
+    repo_id: str
+    access_token: str = None
+    allow_patterns: list[str] = None
+    ignore_patterns: list[str] = None
+    download_dir: str = field(default="/workspace/datasets")
```

Comment:
Do we want to give user control where we are going to download the data if we always download the data in the `/workspace` directory: https://github.com/kubeflow/training-operator/blob/7802b2dc3391c40dadd7e675d778f1bef03285ab/sdk/python/kubeflow/training/api/training_client.py#L174 ?

Reply:
ok

Reply:
@deepanker13 Should we remove this parameter than ?

Reply:
I have made it read only